### ISSUE TITLE
Unexpected value 'Ng2GoogleChartsModule' imported by the module 'MyModule'

### ISSUE THREAD

**Issue by viwell** [gmazzamuto/ng2-google-charts#8]
Hi, I'm getting this error message in version 1.0.2: Unexpected value 'Ng2GoogleChartsModule' imported by the module 'MyModule'

It cannot be imported as:

```ts
import { Ng2GoogleChartsModule } from "ng2-google-charts";
```

now, in order to make it works, It needs to be imported like:

```ts
import { Ng2GoogleChartsModule } from "ng2-google-charts/dist";
```

Any ideas about it?

**Comment by gmazzamuto** [comment]
my fault, I have just published version 1.0.4. Can you please verify it works?

**Comment by viwell** [comment]
It works like a charm, big thanks!
